trainingMaquetado/Maquetado.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>   
+    <title>Cheese Guardian</title>
+    <link rel="stylesheet" href="Maquetado.css">
```

Comment:
use lower case for file names, also it's preferred that this file is named `index.html` since files with that name are the default files to be loaded when they are in a server